yt/utilities/command_line.py

```diff
@@ -1617,65 +1622,115 @@ def __call__(self, args):
         print(r.text)
 
 
-class YTConfigGetCmd(YTCommand):
+class YTConfigLocalConfigMixin:
+    def load_config(self, args):
+        from yt.config import GLOBAL_CONFIG_FILE, LOCAL_CONFIG_FILE
+        from yt.utilities.configure import CONFIG
+
+        if "local" in args and args.local:
+            if LOCAL_CONFIG_FILE is None:
+                LOCAL_CONFIG_FILE = os.path.join(
+                    os.path.abspath(os.path.curdir()), "yt.toml"
+                )
```

Comment:
I'm having mixed feelings here. Could you explain why this isn't/should be part of `LOCAL_CONFIG_FILE`'s definition in the module it originates from ? I'm trying to disentangle the different use cases but I can't figure this out in a satisfying way.

Reply:
I think this should go once we settle on whether we want to load both global and local config files or not. This could be moved into the YTConfig object.